tests/bluetooth/tester/src/btp_csis.c

```diff
@@ -61,6 +63,15 @@ static uint8_t csis_get_member_rsi(const void *cmd, uint16_t cmd_len,
 	return BTP_STATUS_VAL(err);
 }
 
+static uint8_t csis_set_sirk_type(const void *cmd, uint16_t cmd_len, void *rsp, uint16_t *rsp_len)
+{
+	LOG_DBG("Change SIRK type to encrypted");
+
+	enc_sirk = true;
+
+	return BTP_STATUS_SUCCESS;
+}
```

Comment:
Right, this will only be used to set SIRK type to encrypted and can't be used to switch types back and forth. IMO there is no need to implement it that way, because only two test cases (CSIS/SR/SP/BV-08-C and CSIP/SR/SP/BV-03-C) require CSIS IUT to expose encrypted SIRK. And it works for former, the latter still has some other issues that I'm working on.

Reply:
@Thalley actually I ended up adding type parameter to command...